projects/igniteui-angular/src/lib/time-picker/time-picker.component.ts

```diff
@@ -84,55 +97,195 @@ export class IgxTimePickerComponent implements ControlValueAccessor, OnInit, OnD
         }
     }
 
+    /**
+     * An accessor that returns the value of the `_value` property.
```

Comment:
Just "Returns the value" is enough.  No need to mention the private _value holding the actual value.